Stylesheets/components/avtar.css

```diff
@@ -0,0 +1,76 @@
+@import url("/stylesheets/main.css");
+
+.center {
```

Comment:
The class name and what class is doing dont make sense. If you wanted to center content with in a given space instead of making it stick to edges

```css
margin: 0 auto;
width: 90vw;
```

Reply:
yes it was mistake i forget to delete. now i deleted